cypress/package.json

```diff
@@ -3,6 +3,7 @@
     "name": "e2e",
     "version": "0.1.0",
     "scripts": {
+        "serve": "node ./infinitStart.js",
```

Comment:
Why is this needed ? Looks irrelevant to this PR

Reply:
Are you aware of the npm commands inside the root `package.json`. If you can't use make, you can:

launch a terminal and start `yarn run-simple`

launch another terminal and start `yarn test-e2e-local`

Can you remove the new ones?

Reply:
sure